src/miniflask/miniflask.py

```diff
@@ -41,10 +40,9 @@ def get_default_args(func):
 def registerPredefined(modules_avail):
     for m in ["modules", "events", "info", "settings", "definitions"]:
         module_name_id = 'miniflask.' + m
-        importname = 'miniflask.modules.' + m
+        importname = 'miniflask.' + m
```

Comment:
This also seems related to #114?

Reply:
Yes, this is correct. As above, we could push this to master, however, this is just a cosmetic, namely the visual representation for the `[settings]` module.